### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by lbluque**
## Summary

Flakey test fixes.

* `test_run` in test_sampler.py, failed on occasion because Ewald energy tolerances are within 5e-13.
* `test_orbit_mappings`  in test_clusterspace.py failed on occasion due to `StructureMatcher` being unable to match overly skewed super cells.
* `test_convert_formats` in test_comp_space.py failed to raise Error....

## Checklist

- [X] Code is in the [standard Python style](https://www.python.org/dev/peps/pep-0008/).
      Run [pycodestyle](https://pycodestyle.readthedocs.io/en/latest/) and [flake8](http://flake8.pycqa.org/en/latest/)
      on your local machine.
- [X] Docstrings have been added in the [Google docstring format](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).
      Run [pydocstyle](http://www.pydocstyle.org/en/2.1.1/index.html) on your code.
- [X] Tests have been added for any new functionality or bug fixes.
- [X] All existing tests pass.

### CODE REVIEW

File: tests/test_cofe/test_clusterspace.py [hunk 1]
```diff
@@ -380,20 +369,21 @@ def get_corr(occu, space, matrix):
     # Symmetrically equivalent matrices should give the same correlation
     # vectors on the same structure, when using the default orbit mapping.
     cluster_subspace._supercell_orb_inds = {}
-    for s, c in zip(structures, corrs):
+    for structure, expected in zip(structures, corrs):
+        predicted = cluster_subspace.corr_from_structure(structure)
```
Comment (qchempku2017) [inline tests/test_cofe/test_clusterspace.py:373]: Why remove supercell matrix here?

**Review by qchempku2017 — COMMENTED** [review]
Could you please tell we whether you changed the super-cell matrix for testing correlation from structure? If you did, where? Thank you.

File: tests/test_cofe/test_clusterspace.py (hunk 1 above)
Comment (lbluque) [reply tests/test_cofe/test_clusterspace.py:373]: the different supercell matrices are passed below. We can remove the one where no sc matrix is passed and that will prevent possible errors from structures that do not match

**Review by qchempku2017 — APPROVED** [review]
Thank you for the explanation! Approved.

**Pull request merged**